core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -700,6 +758,58 @@ oppia.factory('explorationStatesService', [
         return (
           validatorsService.isValidStateName(newStateName, showWarnings));
       },
+      getStateContentMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'content');
+      },
+      saveStateContent: function(stateName, newContent) {
+        saveStateProperty(stateName, 'content', newContent);
+      },
+      getStateParamChangesMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'param_changes');
+      },
+      saveStateParamChanges: function(stateName, newParamChanges) {
```

Comment:
Please add a TODO that these need unit tests if you prefer to avoid the additional review round.